### DISCUSSION TITLE
Improving openssf scorecard

### DISCUSSION THREAD

**Discussion by ramrattle** [keylime/keylime#1567] [Ideas]
I just ran openssf scorecard for keylime, can we create some PR's to improve few of these

https://scorecard.dev/viewer/?uri=github.com%2Fkeylime%2Fkeylime

**Comment by THS-on** [comment]
This can probably added to https://github.com/keylime/keylime/issues/929. Do you have any specific one that you want to tackle?

**Reply by THS-on** [reply]
Also note that we do not really use the discussions tab. Probably raising an issue or asking on the slack gets more responses.
